quantize.py

```diff
@@ -20,13 +20,21 @@
 from devices import device
 
 CONV_SCALE_BITS = 8
+CONV_DEFAULT_WEIGHT_BITS = 8
 FC_SCALE_BITS = 8
 FC_CLAMP_BITS = 8
 
-DEFAULT_SCALE = .85
+DEFAULT_SCALE = 1.0
```

Comment:
is the scale used in QAT as well?

Reply:
If QAT is disabled, the same 0.85 default is expected.

Reply:
Good point. Checking with IST.

Reply:
Going back to .85 for the default.